engine/src/main/java/org/pentaho/di/repository/filerep/KettleFileRepository.java

```diff
@@ -958,7 +958,6 @@ public JobMeta loadJob( String jobname, RepositoryDirectoryInterface repdir, Pro
     jobMeta.setRepository( this );
     jobMeta.setMetaStore( MetaStoreConst.getDefaultMetastore() );
 
-    readDatabases( jobMeta, true );
```

Comment:
Why is this removed?

Reply:
We no longer need to load shared objects from the repository into Metas. Shared objects from the repository are now accessed through the DefaultBowl.